src/Views/listings/search.phtml

```diff
@@ -19,7 +19,7 @@ $title = "Search page";
             <div class="form-check">
                 <form action="/listings/search"  method="GET">
                     <h6>Trade Method</h6>
-                    <input type="hidden" value="<?=$params['query']?>" name="query">
+                    <input type="hidden" value="<?=isset($params['query']) ?? $params['query'] ?>" name="query">
```

Comment:
isset shouldn't be used here, since it never returns null the right hand side of the null coalescing operator will never be returned. $params['query'] should be on the left side here, and just '' of the right, since if it isn't set no value is needed.